### PR TITLE
BZ1880421: Added warning for --filter-by-os parameter.

### PR DESCRIPTION
**Pull request by jonquilwilliams**
https://bugzilla.redhat.com/show_bug.cgi?id=1880421

QA and peer review needed. Alex, I am having trouble with formatting that Warning - it's changing the numbering on these options. Before i send this for QA review, I'll need to work with you to fix this.

### CODE REVIEW

File: modules/olm-restricted-networks-configuring-operatorhub.adoc [hunk 1]
```diff
@@ -94,6 +94,9 @@ the `--insecure` flag.
 architectures and operating systems, you can filter by architecture and
 operating system to mirror only the images that match. Valid values are
 `linux/amd64`, `linux/ppc64le`, and `linux/s390x`.
++
+CAUTION: Do not use `--filter-by-os` to mirror the entire OLM catalog because it changes the ManifestList digest in the target registry for multi-arch images and breaks the catalog consistency. Specifically, the Operator will not be able to find sha256 digests.
```
Comment (adellape) [inline modules/olm-restricted-networks-configuring-operatorhub.adoc:98]: s/sha256/SHA-256/

Comment (jonquilwilliams) [reply modules/olm-restricted-networks-configuring-operatorhub.adoc]: Thanks, Alex. Your edits have been incorporated.

**Comment by jonquilwilliams** [comment]
I am closing this pull request because it is stale and the documentation has changed a lot since this was first done. Going to open a new PR to replace this.

**Pull request closed without merging**